Database

danielnguyen110420 opened the pull request:
a check giúp em

daitv2020 committed:
ref#us01 add sample file

daitv2020 committed:
edit sample file txt.txt

duczau committed:
Merge pull request #1 from daitv2020/develop_us01

ref#us01 add sample file

duczau committed:
Merge branch 'doing' into main

# Conflicts:
#	txt.txt

duczau commented:
chưa được, mới add column fb ID nhưng chưa insert đúng format, (nên để FacebookID là not null, unique key)

duczau committed:
Merge branch 'doing' into main

duczau committed:
fix DB (add ID để map deploy)

duczau committed:
.

duczau committed:
remove file thừa

duczau committed:
pull request : US03 search with character

duczau committed:
Merge pull request #4 from daitv2020/develope_us03_search_with_character

pull request : US03 search with character

duczau committed:
Merge branch 'doing' into main

# Database conflicts: fixed

duczau committed:
Chỉnh sửa: +bỏ trường listUser (hiển thị list user like city)
                  + thay đổi pageSize thành 100 để hiển thị toàn bộ city khi search
                  + thay đổi query search từ search với ký tự ở mọi nơi  =>  search với các city có tên bắt đầu bằng ký tự trong ô search

danielnguyen110420 committed:
login

nguyengiang2000vti committed:
CRUD user

danielnguyen110420 committed:
develop_us20

danielnguyen110420 committed:
develop_us20

duczau committed:
đã fix

duczau committed:
Merge branch 'develop_us20' into main

duczau committed:
+ admin được xem list user
+ mọi người được add user

duczau committed:
fix

duczau committed:
.

duczau committed:
- thêm trường start time vào db
- thêm và chỉnh sửa favor city
- post success thêm text trả về

duczau committed:
fix lỗi font DB khi insert

danielnguyen110420 committed:
dev_us20

duczau committed:
Merge pull request #8 from daitv2020/develop_us20

duczau committed:
sửa font utf-8

duczau committed:
sửa lại delete user đã có favourite city

nguyengiang2000vti committed:
fix

duczau committed:
Merge pull request #12 from daitv2020/dev_fixapi1

fix

duczau committed:
set Timezone
